react-renderer/src/engine/PropagateUpdate.ts

```diff
@@ -41,7 +41,7 @@ export const insertPending = (state: State) => {
       .map((p: Path) => [p, findShapeProperty(state.shapes, p).updated])
       // .updated is from `collectLabels` updating pending properties
       .reduce(
-        (trans: Translation, [path, v]: [Path, Value<number>]) =>
```

Comment:
Can you say why the types were made more permissive here? I'd prefer them to have more specific types if possible

Reply:
The version bump actually made typechecking more rigorous. I think it has to do with the usage of `zip`. Same thing is true in the Optimizer module. Thanks for noting this. We will need to fix a couple of them before merging this PR